src/DocumenterCitations.jl

```diff
@@ -1,7 +1,28 @@
 module DocumenterCitations
 
+using Documenter
+using Documenter.Anchors
+using Documenter.Builder
+using Documenter.Documents
+using Documenter.Selectors
+using Documenter.Utilities
+using Documenter.Expanders
+
+using Markdown
+using Bibliography
+using Bibliography: xnames, xyear, xlink, xtitle, xin
+
+export CitationBibliography
+struct CitationBibliography <: Documenter.Plugin
+    bib # dictionary
+end
```

Comment:
```suggestion

struct CitationBibliography{B} <: Documenter.Plugin
    bib :: B
end
```

Might be good to leave it concretely typed in case we/someone decides to use something that's not a `Dict` but looks and works like a `Dict`? Otherwise we can use `bib :: Dict`.

Reply:
Yeah, it looks like it's expected to behave like a dict. I think `bib::Dict` should work fine. I was running into issues when using concrete types (the keys of the dict are `typeof(CitationBibliography(...))`, which includes the parameters), I think there's a way to make it work, but I wasn't sure it was worth the effort since the generic form works.

Reply:
I'll make it `bib::Dict`

Reply:
Ah sounds good then!